### DISCUSSION TITLE
A Small Intro of Myself !!

### DISCUSSION THREAD

**Discussion by abhishek-vs** [community/community#20260] [Other Feature Feedback, Questions, & Ideas]
Hello everyone! 👋
I am Abhishek V S, currently pursuing my B.tech Degree in PRESIDENCY UNIVERSITY in the branch of Information Science and Engineering. Nice meeting you all. I will be looking forward to interact, collaborate and get along with you in future.

Thank you,

regards, 
Abhishek V S

**Comment by radar-blip** [comment]
Greetings from USA and welcome to GitHub!
